### PR TITLE
DEV2-3321 allow smtp connection

### PR DESCRIPTION
**Pull request by amirbilu**

### CODE REVIEW

**Review by assapir — APPROVED** [review]

File: charts/tabnine-cloud/templates/auth/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Kind of a temp patch to allow smtp connection
+{{- if .Values.global.networkPolicy.enabled }}
```
Comment (yanir-codota) [inline charts/tabnine-cloud/templates/auth/networkpolicy.yaml:2]: is this applied only for the auth as we discussed?

Comment (yanir-codota) [reply charts/tabnine-cloud/templates/auth/networkpolicy.yaml:2]: nevermind. I see it's under auth directory
sorry,

**Review by yanir-codota — APPROVED** [review]
LGTM

File: charts/tabnine-cloud/templates/auth/networkpolicy.yaml (hunk 1 above)
Comment (amirbilu) [reply charts/tabnine-cloud/templates/auth/networkpolicy.yaml:2]: Yes

**Pull request merged**